integration/gateway/gateway_test.go

```diff
@@ -238,13 +238,11 @@ var _ = Describe("GatewayService", func() {
 				Signature: signature,
 			}
 
-			actualStatus, err := gatewayClient.CommitStatus(ctx, signedStatusRequest)
+			status, err := gatewayClient.CommitStatus(ctx, signedStatusRequest)
 			Expect(err).NotTo(HaveOccurred())
 
-			expectedStatus := &gateway.CommitStatusResponse{
-				Result: peer.TxValidationCode_VALID,
-			}
-			Expect(proto.Equal(actualStatus, expectedStatus)).To(BeTrue(), "Expected\n\t%#v\nto proto.Equal\n\t%#v", actualStatus, expectedStatus)
+			Expect(status.Result).To(Equal(peer.TxValidationCode_VALID))
+			Expect(status.BlockNumber).To(BeNumerically(">", 0))
```

Comment:
Better to use the exact block number here.

Reply:
How do we know what the block number will be without introducing a fragile dependency on other integration tests and their ordering?

Reply:
Not sure I understand the scope when you say other integration tests but before `Describe("CommitStatus"`, it sets up a fresh network...no?

Reply:
My worry is, what is the block number I should pick? And are we guaranteed that will not change in the future, either because of changes to Fabric implementation or because somebody comes along and adds another test case to this Describe block? I think I'd prefer to test that some sane block number came back rather than deliver fragile tests that may break in the future for reasons not connected to this particular test. It's a moot point for now as we're not happy with the approach for obtaining the block number so I'll remove the assertion for now

Reply:
I've changed the test to check the commit status of two subsequent transactions so I can test that the block number of the second commit is one higher than the block number of the first